src/main/java/com/denizenscript/depenizen/bukkit/events/jobs/JobsJobsExpGainScriptEvent.java

```diff
@@ -28,6 +30,8 @@ public class JobsJobsExpGainScriptEvent extends BukkitScriptEvent implements Lis
     // <context.job> Returns a JobsJobTag of the job that the player is gaining exp for.
     // <context.experience> Returns an ElementTag(Decimal) of the amount of exp the player will earn.
     // <context.action> Returns an ElementTag of the name of the action being paid for, which can be any of the strings from: <@link url https://github.com/Zrips/Jobs/blob/master/src/main/java/com/gamingmesh/jobs/container/ActionType.java>.
+    // <context.entity> Returns an EntityTag of the entity involved with this event, if applicable.
+    // <context.block> Returns a LocationTag of the block involved with this event, if applicable.
```

Comment:
The meta here looks a little vague - what are the entity/block doing in the context of each event?

Reply:
Jobs itself isn't very clear about this.

The entity is the target of the action, so if i killed a pig and earned money/exp, the entity would be the pig i killed.
It's similar for block, where the location is the block i would have modified.

I'm not really sure how else to word it in the meta.

Reply:
Acting on feedback from discord, I've added an example and slightly changed the wording on the reviewed lines